src/Xamarin.Android.Build.Tasks/Properties/Resources.resx

```diff
@@ -434,6 +434,14 @@ In this message, the term "binding" means a piece of generated code that makes i
     <comment>The following are literal names and should not be translated: .NET, Xamarin.Android.
 {0} - The file name such as 'Foo.dll.config'</comment>
   </data>
+  <data name="XA1026" xml:space="preserve">
+    <value>Using 'aapt' is deprecated in favor of 'aapt2'. Please enable 'Use incremental Android packaging system (aapt2)' in the Visual Studio project property pages or edit the project file in a text editor and set the 'AndroidUseAapt2' MSBuild property to 'true'.</value>
+    <comment>The following are literal names and should not be translated: aapt, aapt2, Android, AndroidUseAapt2.</comment>
+  </data>
+  <data name="XA1026_dotnet" xml:space="preserve">
+    <value>Using 'aapt' is not supported in Xamarin.Android projects that target .NET 5 or higher. Please enable 'Use incremental Android packaging system (aapt2)' in the Visual Studio project property pages or edit the project file in a text editor and set the 'AndroidUseAapt2' MSBuild property to 'true'.</value>
+    <comment>The following are literal names and should not be translated: aapt, aapt2, Android, AndroidUseAapt2.</comment>
+  </data>
```

Comment:
Since `'aapt'` and `'aapt2'` aren't argument values for the MSBuild property, while `'true'` is, maybe:

```xml
  <data name="XA1026" xml:space="preserve">
    <value>Using AAPT is deprecated in favor of AAPT2. Please enable 'Use incremental Android packaging system (aapt2)' in the Visual Studio project property pages or edit the project file in a text editor and set the 'AndroidUseAapt2' MSBuild property to 'true'.</value>
    <comment>The following are literal names and should not be translated: AAPT, AAPT2, Android, AndroidUseAapt2, true.</comment>
  </data>
  <data name="XA1026_dotnet" xml:space="preserve">
    <value>Using AAPT is not supported in Xamarin.Android projects that target .NET 5 or higher. Please enable 'Use incremental Android packaging system (aapt2)' in the Visual Studio project property pages or edit the project file in a text editor and set the 'AndroidUseAapt2' MSBuild property to 'true'.</value>
    <comment>The following are literal names and should not be translated: AAPT, Android, AndroidUseAapt2, true.</comment>
  </data>
```